Jenkinsfile

```diff
@@ -94,7 +94,7 @@ void runTest(String TEST_NAME, String CLUSTER_PREFIX) {
             FILE_NAME = "${env.GIT_BRANCH}-${env.GIT_SHORT_COMMIT}-$TEST_NAME-gke-${env.PLATFORM_VER}"
             popArtifactFile("$FILE_NAME")
 
-            timeout(time: 30, unit: 'MINUTES') {
+            timeout(time: 90, unit: 'MINUTES') {
```

Comment:
As agreed with @tplavcic , in order for tests to pass this timeout had to be increased. Also this is the timeout we have on our other operators, so it can stay in here as well.